graphite/graphite.go

```diff
@@ -94,7 +94,7 @@ func (p *Parser) Parse(line string) (influxdb.Point, error) {
 		return influxdb.Point{}, err
 	}
 
-	timestamp := time.Unix(0, unixTime*int64(time.Millisecond))
+	timestamp := time.Unix(unixTime, 0)
```

Comment:
Yeah, I had this vague memory of seeing Graphite data with fractional timestamps. Thanks for looking into it @jwilder